### DISCUSSION TITLE
skiping all target

### DISCUSSION THREAD

**Discussion by elite10833** [projectdiscovery/nuclei#5708] [Q&A]
whenever i try to run nuclei it skip all the target and show
[INF] Skipped xxxxxxxxxxxxxxxxx:443 from target list as found unresponsive 30 times
[INF] Skipped xxxxxxxxxxxx:443 from target list as found unresponsive 30 times

i have tried using one target at a time but the results are same

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
